protocol/avdl/secret_ui.avdl

```diff
@@ -37,13 +37,15 @@ protocol secretUi {
 
   string getPaperKeyPassphrase(int sessionID, string username);
 
-  record SecretStorageFeature {
+  record Feature {
     boolean allow;
+    boolean respond; /* if we want the value back */
     string label;
   }
 
   record GUIEntryFeatures {
-    SecretStorageFeature secretStorage;
+    Feature storeSecret;
+    Feature showTyping;
```

Comment:
what i'll do is if allow is there, it'll send it over the wire, with default checked as a value